src/scraping/collector.py

```
@@ -0,0 +1,123 @@
+"""
+MVP Data Collection using BrightData
+"""
+from typing import List, Dict, Any
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)